tailfs/birthtiming.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package tailfs
+
+import (
+	"context"
+	"io/fs"
+	"os"
+	"time"
+
+	"github.com/djherbis/times"
+	"github.com/tailscale/xnet/webdav"
+)
+
+// birthTimingFS extends a webdav.FileSystem to return FileInfos that implement
+// the BirthTimer interface.
```

Comment:
It's interesting, as I actually would be more confused by calling it `BirthTimerFS` because the FS is not a BirthTimer, it just returns files that are.

Reply:
Fixed comment